config/management/src/secure_backend.rs

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::error::Error;
+use libra_config::config::{self, OnDiskStorageConfig, VaultConfig};
+use std::{
+    collections::HashMap,
+    convert::{TryFrom, TryInto},
+    path::PathBuf,
+};
+
+/// SecureBackend is a parameter that is stored as a colon separated key/value pairs. The only
```

Comment:
nit: "that is stored as colon separated.."?

Reply:
and it is actually semi-colon lol